pyproject.toml

```diff
@@ -92,6 +92,7 @@ select = [
     "E",    # pycodestyle
     "F",    # pyflakes
     "I",    # isort
+    "UP",   # pyupgrade
```

Comment:
Cool, looks like it automatically sets the [target-version](https://docs.astral.sh/ruff/settings/#target-version) from `project.requires-python`.